lib/SIL/SILCoverageMap.cpp

```diff
@@ -42,13 +42,10 @@ SILCoverageMap::create(SILModule &M, StringRef Filename, StringRef Name,
   CM->Expressions = M.allocateCopy(Expressions);
 
   // Assert that this coverage map is unique.
-  assert(llvm::none_of(M.coverageMaps,
-                       [&](const SILCoverageMap &OtherCM) {
-                         return OtherCM.PGOFuncName == CM->PGOFuncName;
-                       }) &&
+  assert(M.coverageMaps.find(CM->PGOFuncName) == M.coverageMaps.end() &&
```

Comment:
A better idiom than `assert(foo.find()); foo[key] = value` is

`auto result = foo.insert(std::make_pair(key, value));
assert(!result.second);`

Reply:
Sounds good, thanks.